src/lib.rs

```diff
@@ -109,7 +109,7 @@ impl Troffable for Section {
     fn render(&self) -> String {
         let mut res = String::new();
 
-        writeln!(&mut res, ".SH {}", self.title.to_uppercase()).unwrap();
+        writeln!(&mut res, ".SH \"{}\"", self.title.to_uppercase()).unwrap();
```

Comment:
This will fail if the title contains quote characters. I'm not sure how to fix that.

Reply:
Oddly, I'm not seeing any quote escaping in asciidoctor, only left/right quotes.  I wonder if that is an artifact of them only using left/right quotes

https://github.com/asciidoctor/asciidoctor/blob/main/lib/asciidoctor/converter/manpage.rb#L716

Reply:
For some more information on escaping, check out https://www.ibm.com/docs/en/aix/7.2?topic=t-troff-command (search for mm Strings)

Reply:
I've been reading the [groff(7)](https://manpages.debian.org/bullseye/groff/groff.7.en.html), and I can't see any way to escape quotes in control lines or macro arguments.